app/models/ems_event.rb

```diff
@@ -58,14 +58,46 @@ def self.group_name(group)
     group[:name]
   end
 
-  def self.add_queue(meth, ems_id, event)
-    MiqQueue.submit_job(
-      :service     => "event",
-      :target_id   => ems_id,
-      :class_name  => "EmsEvent",
-      :method_name => meth,
-      :args        => [event],
-    )
+  def self.artemis_client
+    require "manageiq-messaging"
+    # FIXME: logger should be set at a global level
+    ManageIQ::Messaging.logger = _log
+    @artemis_client ||= begin
+      queue_settings = Settings.workers.worker_base.queue_worker_base.event_handler
```

Comment:
This probably should not be in Settings, but instead just baked into env vars like we do with other services for the podified world  cc @carbonin

Reply:
@carbonin is there already something using ENV vars instead of settings?
Otherwise I could "mock" it by moving that code to a module that prefers ENV vars over Settings